core/modules/node/node.install

```diff
@@ -313,6 +313,12 @@ function node_schema() {
  * Implements hook_install().
  */
 function node_install() {
+  $content_types = array();
+  foreach (node_type_get_types() as $type => $content_type) {
+    $content_types[$type] = $type;
+  }
+  config_set('search.settings', 'content_types', $content_types);
+
```

Comment:
I feel that this should be in `search.install` instead; and although this handles the installation (so new sites basically), we'd also need an update hook, to add all content types into the indexable list for existing sites.

Reply:
This should be deleted actually since node types get created in `standard.profile` which installs last, so this wont work.

Reply:
I think the enabling of indexing on new content types should be moved to [hook_node_type_insert()](https://api.backdropcms.org/api/backdrop/core%21modules%21node%21node.api.php/function/hook_node_type_insert/1), that would make this work with all profiles and on the creation of new content types.

We'd still need an update hook to enable all the content types on existing sites.